_includes/head.html

```diff
@@ -6,7 +6,15 @@
     <title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
     <meta name="description" content="{{ site.description }}">
 
+    {% if page.canonical_url and page.canonical_url != page.url %}
+    <link rel="canonical" href="{{ page.canonical_url | replace:'index.html','' }}">
```

Comment:
Nipick: space after colon.

Same below.

Reply:
Good point. Fixed.

Reply:
Should we add a space after the commas too (in a few cases)?

Reply:
Sounds good!